RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate ScreenLockConfigView to Typescript

## Proposed changes

Migrate ScreenLockConfigView to Typescript

## Issue(s)	


## How to test or reproduce

- When logged, go to `Settings -> Security and privacy -> Screen lock`
- Then, this is the view to check the behavior.

## Screenshots

## Types of changes




## Checklist



## Further comments
